src/transaction/guards.ts

```diff
@@ -35,9 +35,12 @@ export const isTxInput = (
 export const isTxOutput = (
   test: any,
 ): test is TxOutput => {
-  if (!Array.isArray(test) || test.length !== 2) return false
-  const [address, coins] = test
-  return Buffer.isBuffer(address) && isLovelace(coins)
+  if (Array.isArray(test) && test.length === 2) {
```

Comment:
I don't like the name "test" for a variable that does not represent a test. (But I can live with that. Generally, I would suggest to avoid the word "test" for anything not being a test with the meaning like in a unit test. Nothing good would come out of it, just confusion.)

Also, "coins" does not sound appropriate for an object that might legally be something more complex.

Reply:
I agree, I don't like the `test` name for everything that comes to guards as well, we did it initially and kind of just went with it. I ll refactor this before the official release. Thanks. 
Regarding the `coin`, I missed that one as I was renaming it to amount at all places, thnx, fixed.

Reply:
Refactored from `test` to `value`